app/controllers/UpfrontPaymentController.scala

```diff
@@ -112,18 +121,37 @@ class UpfrontPaymentController @Inject() (
 
 object UpfrontPaymentController {
 
-  def canYouMakeAnUpfrontPaymentQuestionForm: Form[String] = Form(
-    mapping(
-      "CanYouMakeAnUpFrontPayment" -> nonEmptyText
-    )(identity)(Some(_))
-  )
+  sealed trait CanUpfrontPaymentFormValue extends enumeratum.EnumEntry {
```

Comment:
```suggestion
  sealed trait CanPayUpfrontFormValue extends enumeratum.EnumEntry {
```
that goes a bit better with the `CanPayUpfront` root model